src/librawspeed/decompressors/FujiDecompressor.cpp

```diff
@@ -0,0 +1,1097 @@
+/*
+    FujiDecompressor - Decompress Fujifilm compressed RAF.
+
+    Copyright (C) 2016 Alexey Danilchenko
+    Copyright (C) 2016 Alex Tutubalin
+    Copyright (C) 2017 Uwe Müssel
+
+    This library is free software; you can redistribute it and/or
+    modify it under the terms of the GNU Lesser General Public
+    License as published by the Free Software Foundation; either
+    version 2 of the License, or (at your option) any later version.
```

Comment:
```diff
+    License as published by the Free Software Foundation; either
+    version 2 of the License, or (at your option) any later version.
```
Hmm, where did you get that?
https://github.com/LibRaw/LibRaw/commit/aeab4d8ea58f85050d1253bdc41f51baa79ffe65#diff-03b018f1deeb8fc38d2c927f09465635R11 says:
```diff
+LibRaw is free software; you can redistribute it and/or modify
+it under the terms of the one of three licenses as you choose:
+
+1. GNU LESSER GENERAL PUBLIC LICENSE version 2.1
+   (See file LICENSE.LGPL provided in LibRaw distribution archive for details).
+
+2. COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.0
+   (See file LICENSE.CDDL provided in LibRaw distribution archive for details).
+
```
(2 vs 2.1)